### ISSUE TITLE
nbb no longer finds local namespaces after 1.0.140

### ISSUE THREAD

**Issue by aisamu** [babashka/nbb#279]
# Version

```shell
$> npm ls nbb
nbb@1.0.140
```

> Also tested on `141` and `142` and the same happened.

# Platform

```shell
$> uname -a
Darwin aisambp-stedi 21.6.0 Darwin Kernel Version 21.6.0: Wed Aug 10 14:28:23 PDT 2022; root:xnu-8020.141.5~2/RELEASE_ARM64_T6000 arm64 arm Darwin
```

```shell
$> npm --version && node --version
8.19.2
v18.12.1
```

# Problem

`nbb` doesn't find local namespaces.

# Reproduction

## Project shape

```clojurescript
;; ./test.cljs
(ns test
  (:require [foo.bar :as fb]))

(prn *file*)
(prn "TEST")
```

```clojurescript
;; ./foo/bar.cljs
(ns foo.bar)

(prn *file*)
(prn "TEST2")
```

### `< 1.0.140`

```
$> npm i nbb@1.0.139 && nbb test.cljs

up to date, audited 310 packages in 2s

56 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
"/<path>/foo/bar.cljs"
"TEST2"
"<path>/test.cljs"
"TEST"
```

### `>` 1.0.140=

```
$> npm i nbb@1.0.140 && nbb test.cljs

changed 1 package, and audited 310 packages in 502ms

56 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
----- Error --------------------------------------
Message:  Could not find namespace: foo.bar
Could not find namespace: foo.bar
```

Happy to provide further details/examples!

Thanks again for `nbb`!

**Comment by borkdude** [comment]
Should be fixed with 1.0.144 being pushed on CI now

**Comment by aisamu** [comment]
Whoa, that's fast! Thank you 🥳
